test/test-utils.js

```diff
@@ -0,0 +1,7 @@
+import * as jestDOM from '@testing-library/jest-dom'
+
+expect.extend(jestDOM)
+
+export * from '@testing-library/jest-dom'
+export * from '@testing-library/react'
+export { faker } from '@faker-js/faker'
```

Comment:
Smush as much of the boilerplate code stuff into a single shared file as possible.  One thing I did, I eventually accrued like 30+ lines of boilerplate in each test file and it was really, really gross 🤢  I just didn't know about this approach.